.github/workflows/python.yml

```diff
@@ -58,15 +58,13 @@ jobs:
         with:
           components: rustfmt, clippy
 
-      - name: Install Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: '20'
+      - uses: pnpm/action-setup@v4
```

Comment:
steps aligned with the other workflows.
`pnpm/action-setup@v4` and then `actions/setup-node@v5`